docs/whats-new/dotnet-2020-03.md

```diff
@@ -0,0 +1,114 @@
+---
+title: ".NET docs: What's new for March 2020"
+description: "What's new in .NET docs for March 2020."
+ms.date: 03/30/2020
+---
+
+# .NET docs: What's new for March 2020
+
+Welcome to what's new in .NET docs for March 2020. This article lists some of the major changes to docs during this period.
+
+## .NET Core
+
+### New articles
+
+- [Breaking changes for migration from version 3.1 to 5.0](../core/compatibility/3.1-5.0.md) - Document SignalR breaking change in ASP.NET Core 5.0
+- [.NET Core documentation](../core/index.yml) - .NET Core landing page
+- [dotnet nuget add source](../core/tools/dotnet-nuget-add-source.md) - docs for `dotnet nuget add source` and related
+- [dotnet nuget disable source](../core/tools/dotnet-nuget-disable-source.md) - docs for `dotnet nuget add source` and related
+- [dotnet nuget enable source](../core/tools/dotnet-nuget-enable-source.md) - docs for `dotnet nuget add source` and related
+- [dotnet nuget list source](../core/tools/dotnet-nuget-list-source.md) - docs for `dotnet nuget add source` and related
+- [dotnet nuget remove source](../core/tools/dotnet-nuget-remove-source.md) - docs for `dotnet nuget add source` and related
+- [dotnet nuget update source](../core/tools/dotnet-nuget-update-source.md) - docs for `dotnet nuget add source` and related
+
+## C# language
+
+### New articles
+
+- [switch expression (C# reference)](../csharp/language-reference/operators/switch-expression.md) - Add switch expression to language reference
+
+### Updated articles
+
+- [Extension Methods (C# Programming Guide)](../csharp/programming-guide/classes-and-structs/extension-methods.md) - Add rules for using ref and in on extension methods
+
+## .NET Framework
+
+### New articles
+
+- [.NET Framework documentation](../framework/index.yml) - Revamp .NET Framework landing page
```

Comment:
Should we call out the new architecture landing page too?

Reply:
Yes. When did that go live? I would have thought it would have gotten picked up.

Reply:
A week ago - https://github.com/dotnet/docs/pull/17503.

Reply:
Added. I'll check the code to see why this wasn't included.